xcm/xcm-emulator/README.md

```diff
@@ -0,0 +1,5 @@
+# xcm-emulator
+
+xcm-emulator uses relay chain and parachain runtimes.
+Users could plug in Kusama, Statemine, Polkadot and other pre-configured runtimes.
+With up-to-date chain specs, it's able to verify if specific XCM messages work in live networks.
```

Comment:
Not OK for documentation.

Reply:
Extended the documentation to point out that xcm-simulator also exists and when to use one over the other.